cves/CVE-2023-35390.yaml

```diff
@@ -0,0 +1,11 @@
+affectedPackages:
+  - cpe23Uri: cpe:2.3:a:microsoft:microsoft.netcore.app:*:*:*:*:*:*:*:*
+    versionEndExcluding: 6.0.21
```

Comment:
In https://github.com/dotnet/announcements/issues/266
I didn't find affected versions of .Net 6.

Reply:
Under `How do I fix the issue?` it says this:

```
* If you're using .NET 7.0, you should download and install Runtime 7.0.10 or SDK 7.0.110 (for Visual Studio 2022 v17.4) from https://dotnet.microsoft.com/download/dotnet-core/7.0.
* If you're using .NET 6.0, you should download and install Runtime 6.0.21 or SDK 6.0.316 (for Visual Studio 2022 v17.2) from https://dotnet.microsoft.com/download/dotnet-core/6.0.

```